kitchen.yml

```
@@ -240,10 +240,32 @@ suites:
     verifier:
       inspec_tests:
         - path: test/integration/default
-  - name: centos
+  - name: centos8
```

```suggestion
  - name: redhat8
```

Will be required for `oraclelinux-8` as well, so suggest renaming this accordingly (also matches the test pillar name below).